File: CurseCS/Model/Pilot.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace curse.Model
{
    [Serializable]
    public class Pilot : Entity<long>
    {
        public string nume { get; private set; }
        public string numeEchipa { get; private set; }

        public Pilot(string nume, string numeEchipa)
        {
            Id = 0;
            this.nume = nume;
            this.numeEchipa = numeEchipa;
        }
        public Pilot(long id, string nume, string numeEchipa)
        {
            Id = id;
            this.nume = nume;
            this.numeEchipa = numeEchipa;
        }
    }
}

File: Curse/DomainProject/src/main/java/Domain/Pilot.java
package Domain;


import java.io.Serializable;

public class Pilot extends Entity<Long> implements Serializable {
    public String Nume;
    public String NumeEchipa;

    public String getNume() {
        return Nume;
    }

    public void setNume(String nume) {
        this.Nume = nume;
    }

    public String getNumeEchipa() {
        return NumeEchipa;
    }

    public void setNumeEchipa(String numeEchipa) {
        this.NumeEchipa = numeEchipa;
    }

    public Pilot(Long id,String nume, String numeEchipa) {
        Id=id;
        this.Nume = nume;
        this.NumeEchipa = numeEchipa;
    }

    public Pilot(String nume, String numeEchipa) {
        Id= Long.valueOf(-1);
        this.Nume = nume;
        this.NumeEchipa = numeEchipa;
    }

}
